interfaces/cython/cantera/examples/reactors/ic_engine.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+"""
+Simulation of a (gaseous) Diesel internal combustion engine.
+
+The reaction mechanism used is the Madison n-heptane reaction mechansim. It
+can be downloaded from https://www.erc.wisc.edu/chemicalreaction.php
+(Caution: use uppercase species names!)
```

Comment:
Can we either (a) ask for permission to include a version of this mechanism with Cantera or (b) come up with an example that just uses GRI-mech, even if it isn't physically relevant?